js/apps/system/_admin/aardvark/APP/react/src/views/collections/addCollection/AddCollectionForm.tsx

```diff
@@ -0,0 +1,170 @@
+    );
+  }
+  if (!database) return null;
+  const isOneShard = database.isOneShard || window.frontendConfig.forceOneShard;
+  return (
+    <Grid templateColumns={"1fr"} gap="6">
+      <Stack>
+        <Text>Basic</Text>
+        <Divider borderColor="gray.400" />
+        <Grid templateColumns={"1fr 1fr"} columnGap="4" alignItems="start">
+          <InputControl isDisabled={isDisabled} name="name" label="Name" />
+          <SelectControl
+            isDisabled={isDisabled}
+            name={"type"}
+            label="Type"
+            selectProps={{
+              options: [
+                { label: "Document", value: "document" },
+                { label: "Edge", value: "edge" }
+              ]
+            }}
+            tooltip={
+              "Use the Document type to store json documents with unique _key attributes. Can be used as nodes in a graph. <br> Use the Edge type to store documents with special edge attributes (_to, _from), which represent relations. Can be used as edges in a graph."
+            }
+          />
+        </Grid>
+        <Text>Configuration</Text>
+        <Divider borderColor="gray.400" />
+        <Grid
+          templateColumns={"1fr 1fr"}
+          rowGap="5"
+          columnGap="3"
+          marginY="4"
+          maxWidth="full"
+        >
+          {window.App.isCluster && (
+            <>
+              <InputControl
+                isDisabled={
+                  isDisabled || isOneShard || distributeShardsLike.value !== ""
+                }
+                name={"numberOfShards"}
+                label="Number of shards"
+                inputProps={{
+                  type: "number",
+                  min: 1,
+                  max: window.frontendConfig.maxNumberOfShards
+                }}
+                tooltip={
+                  isOneShard
+                    ? undefined
+                    : `The number of shards to create. The maximum value is ${window.frontendConfig.maxNumberOfShards}. You cannot change this afterwards.`
+                }
+              />
+              <CreatableMultiSelectControl
+                isDisabled={isDisabled}
+                name={"shardKeys"}
+                label="Shard keys"
+                selectProps={{
+                  noOptionsMessage: () => "Start typing to add a key"
+                }}
+                tooltip={"The keys used to distribute documents on shards."}
+              />
+              <InputControl
+                isDisabled={
+                  isDisabled ||
+                  isSatellite.value ||
+                  distributeShardsLike.value !== ""
+                }
+                name={"replicationFactor"}
+                label="Replication Factor"
+                inputProps={{
+                  type: "number",
+                  min: window.frontendConfig.minReplicationFactor ?? 1,
+                  max: window.frontendConfig.maxReplicationFactor
```

Comment:
the default value for this seems to be `10` in the old code (L563 - https://github.com/arangodb/arangodb/pull/19427/files#diff-0161c56667ef54767bc5e47f3e7a5c058afb5912f223bf1f02e45b2201f7ca72L563)